app/views/content/non-uk-teachers/international-qualified-teacher-status.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: Get international qualified teacher status (iQTS)
+subcategory: Get international qualified teacher status (iQTS)
```

Comment:
@laurahermionetennant92 the H2 is the same as the title on the card. Maybe it could be something along the lines of how to train to teach without coming to England - but worded a bit better!